types/shelljs/index.d.ts

```diff
@@ -965,6 +966,51 @@ export type ShellString = string & ShellReturnValue;
 
 export type ShellArray = string[] & ShellReturnValue;
 
+export const ShellString: ShellStringFunction;
+
+export interface ShellStringFunction {
```

Comment:
This should be&nbsp;called `ShellStringConstructor` to&nbsp;match [TypeScript’s standard&nbsp;library](https://github.com/Microsoft/TypeScript/blob/42302705f6d4266778324d8282ed743ffa305d33/lib/lib.es5.d.ts#L517-L527):
```suggestion
export interface ShellStringConstructor {
```